dev/bots/test.dart

```diff
@@ -309,6 +309,7 @@ Future<void> _runWebToolTests() async {
           forceSingleCore: true,
           testPaths: <String>[path.join(kTest, '$kWeb$kDotShard', '')],
           enableFlutterToolAsserts: true,
+          perTestTimeout: const Duration(minutes: 3),
```

Comment:
I thought of this after we had already merged your change...

Reply:
thanks!